flask/app/email.py

```diff
@@ -0,0 +1,92 @@
+# using SendGrid's Python Library
+# https://github.com/sendgrid/sendgrid-python
+import json
+import math
+import os
+from datetime import datetime, timedelta
+from typing import Any
+
+from pytz import timezone
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import From, Mail, ReplyTo, SendAt, To
+
+from flask import current_app as app
+
+from typing import Any, Dict
+
+sg = SendGridAPIClient(os.getenv("SENDGRID_API_KEY"))
+
+tz = timezone("EST")
```

Comment:
I'm going to have a look at whether setting the timezone in the container causes a problem with our APIs that return timestamps.

Reply:
It will because our datetime objects are naive, so it's out of scope for this PR. I think this should be set to `America/Toronto` or it might not account for daylight saving?